sbercloud/provider.go

```diff
@@ -107,6 +109,10 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
+			"sbercloud_cce_cluster":                  huaweicloud.ResourceCCEClusterV3(),
+			"sbercloud_cce_node":                     huaweicloud.ResourceCCENodeV3(),
+			"sbercloud_cce_addon":                    huaweicloud.ResourceCCEAddonV3(),
+			"sbercloud_cce_node_pool":                huaweicloud.ResourceCCENodePool(),
```

Comment:
do we support cce_addon and cce_node_pool?

Reply:
fixed